examples/channel-directions/channel-directions.go

```diff
@@ -7,24 +7,26 @@ package main
 
 import "fmt"
 
-// This `ping` function only accepts a channel for sending
+// This `send` function only accepts a channel for sending
 // values. It would be a compile-time error to try to
 // receive on this channel.
-func ping(pings chan<- string, msg string) {
-	pings <- msg
+func send(sendCh chan<- string, msg string) {
+	fmt.Println("send message:", msg)
+	sendCh <- msg
 }
 
-// The `pong` function accepts one channel for receives
-// (`pings`) and a second for sends (`pongs`).
-func pong(pings <-chan string, pongs chan<- string) {
-	msg := <-pings
-	pongs <- msg
+// The `fwd` function accepts one channel for receives
+// (`sendCh`) and a second for sends (`forwardCh`).
+func fwd(sendCh <-chan string, forwardCh chan<- string) {
```

Comment:
While I agree that "pings" and "pongs" may not be optimal terminology, I can't say this is clearer. You use `sendCh` as a send-only channel in one place and as a receive-only channel in another place. This is confusing.

"Forward" channel is also confusing here, as it's not clear what it's supposed to forward.

Reply:
😉given my translation is confusing is a good indication that the terms are not clear enough.

Maybe the function name should maybe be completely different to the channel name 

channel names = `sendOnly`, `receiveOnly`. 

functions = maybe just `a` and `b`? Unless there is a real underlying meaning to them?